packages/lib-classifier/src/store/ClassificationStore.js

```diff
@@ -130,15 +130,23 @@ const ClassificationStore = types
 
     function completeClassification () {
       const classification = self.active
-      // TODO store intervention metadata if we have a user...
-      self.updateClassificationMetadata({
+
+      const metadata = {
         session: sessionUtils.getSessionID(),
         finishedAt: (new Date()).toISOString(),
         viewport: {
           width: window.innerWidth,
           height: window.innerHeight
         }
-      })
+      }
+
+      const feedback = getRoot(self).feedback
+      if (feedback.isActive && feedback.rules) {
+        metadata.feedback = toJS(feedback.rules)
```

Comment:
Looks good, but there should be a test for this in the classification store tests.

Reply:
I've stubbed out a feedback store in the classification tests, and added a test for this line.